src/components/date/date.spec.tsx

```diff
@@ -881,7 +1004,7 @@ describe("Date", () => {
   });
 
   describe("validation", () => {
-    it.each(["error", "warning", "info"])(
+    it.each<keyof ValidationProps>(["error", "warning", "info"])(
```

Comment:
```suggestion
    it.each(["error", "warning", "info"] as const)(
```

**suggestion** (non-blocking): personal preference but I think I prefer this.

Same on lines 1019 and 1045 (not going to bother making the same comment 3 times 😁 )

Reply:
see previous comment